etc/uams/meson.build

```diff
@@ -42,15 +42,16 @@ if have_embedded_ssl
     ssl_links += libatalk
 endif
 
-if have_ssl
+if have_libgcrypt
```

Comment:
I think we can remove the `have_libgcrypt` conditional and assume that this library is always available. Which we can safely do, since setup will fail early in the Meson script if libgcrypt cannot be detected.